includes/class-amp-travel-blocks.php

```diff
@@ -403,4 +404,274 @@ public function render_block_discover( $attributes ) {
+	 * @return string Output.
+	 */
+	public function render_block_travel_featured( $attributes ) {
+		$locations = get_terms( array(
+			'taxonomy'   => 'location',
+			'meta_key'   => 'amp_travel_featured',
+			'meta_value' => 1,
+			'per_page'   => self::$featured_locations_count,
+			'hide_empty' => false,
+		) );
+
+		// The count has to be 6 to fill the grid.
+		if ( count( $locations ) !== self::$featured_locations_count ) {
+			return '';
+		}
+
+		// Sort the terms and get back sorted term arrays.
+		$locations = $this->sort_terms_for_featured_grid( $locations );
+
+		$output = '<section class="travel-featured pt3 relative clearfix">
+						<header class="max-width-2 mx-auto px1 md-px2 relative">
+							<h3 class="travel-featured-heading h1 bold line-height-2 mb2 center">' . esc_html( $attributes['heading'] ) . '</h3>
+						</header>
+						<div class="max-width-3 mx-auto relative">
+						<div class="travel-featured-grid flex flex-wrap items-stretch">';
+
+		// Each grid slot has specific attributes.
+		$location_params = array(
+			array(
+				'width'  => 336,
+				'height' => 507,
+				'color'  => 'blue',
+			),
+			array(
+				'width'  => 264,
+				'height' => 246,
+				'color'  => 'cyan',
+			),
+			array(
+				'width'  => 264,
+				'height' => 264,
+				'color'  => 'orange',
+			),
+			array(
+				'width'  => 276,
+				'height' => 207,
+				'color'  => 'purple',
+			),
+			array(
+				'width'  => 264,
+				'height' => 286,
+				'color'  => 'cornflower',
+			),
+			array(
+				'width'  => 312,
+				'height' => 507,
+				'color'  => 'teal',
+			),
+		);
+
+		foreach ( $locations as $i => $location ) {
+
+			// Start of the first half-grid.
+			if ( 0 === $i ) {
+				$output .= '<div class="col-12 md-col-6 flex items-stretch flex-auto">';
+
+				// Start of the second column of the first half-grid.
+			} elseif ( 1 === $i ) {
+				$output .= '<div class="flex flex-column items-stretch flex-auto">';
+			} elseif ( 3 === $i ) {
+
+				// Start of the second half-grid and third column.
+				$output .= '<div class="col-12 md-col-6 flex items-stretch flex-auto">
+						<div class="flex flex-column items-stretch flex-auto">';
+			}
+
+			$location_img_id     = get_term_meta( $location['term_id'], 'amp_travel_location_img', true );
+			$location_img_src    = wp_get_attachment_image_src( $location_img_id, 'full' );
+			$location_img_srcset = wp_get_attachment_image_srcset( $location_img_id, 'full' );
+
+			$output .= '<a href="' . esc_url( get_term_link( $location['term_id'] ) ) . '" class="travel-featured-tile flex flex-auto relative travel-featured-color-' .
+							esc_html( $location_params[ $i ]['color'] ) . '">
+							<amp-img class="travel-object-cover flex-auto" layout="responsive" width="' .
+							esc_html( $location_params[ $i ]['width'] ) . '" height="' .
+							esc_html( $location_params[ $i ]['height'] ) . '" srcset="' . esc_html( $location_img_srcset ) . '" src="' . esc_url( $location_img_src[0] ) . '""></amp-img>
```

Comment:
Could this line and the line above use `esc_attr()` for all of the attributes? Maybe there was an issue in using `esc_attr()` with the `srcset`.